src/SFA.DAS.ApprenticeCommitments.Api/Controllers/ApprenticeController.cs

```diff
@@ -1,18 +1,26 @@
 using Microsoft.AspNetCore.JsonPatch;
 using Microsoft.AspNetCore.Mvc;
-using SFA.DAS.ApprenticeCommitments.Apis.InnerApi;
+using SFA.DAS.ApprenticeCommitments.Apis.ApprenticeAccountsApi;
 using SFA.DAS.ApprenticeCommitments.Application.Services;
 using System;
 using System.Threading.Tasks;
 
 namespace SFA.DAS.ApprenticeCommitments.Api.Controllers
 {
+    public class TemporaryAccountsResponseReturningApiClient : ResponseReturningApiClient
```

Comment:
I see what you mean, and yes the whole apprentice controller needs to be removed